src/components/Blog/Post/Markdown/styles.module.css

```diff
@@ -359,6 +364,17 @@
       }
 
       .language-dvctable {
+        .token.rows {
```

Comment:
Whoops! Forgot to add the `dvctablehorizontals` css code to the blog markdown css!